src/user/model.rs

```diff
@@ -36,40 +39,55 @@ pub struct UserData {
 
 #[derive(Debug, Serialize, Deserialize, Clone)]
 pub struct SlimUser {
-    pub email: Option<String>,
+    pub user_uuid: Uuid,
```

Comment:
I think that `user_uuid` is a better identifier for the frontend than simply the email.